src/components/security/SecurityScansTab/SecurityScansTab.tsx

```diff
@@ -359,6 +373,16 @@ export const SecurityScansTab = () => {
         )
     }
 
+    if (!scanListLoading && !securityScansResult?.result.securityScans.length && !areFiltersActive) {
```

Comment:
Suggestion: Can wrap as parent condition if(!scanListLoading && !securityScansResult?.result.securityScans.length) since duplicated and based on areFiltersActive can early return GenericEmptyState